diff --git a/nmake.opt b/nmake.opt
index e68f65a..8594b2d 100644
--- a/nmake.opt
+++ b/nmake.opt
@@ -276,9 +276,9 @@ JPEG_SUPPORTED = 1
 !ENDIF
 
 # This will enable 12bit libjpeg - use only with internal jpeg builds.
-!IFNDEF JPEG12_SUPPORTED
-JPEG12_SUPPORTED = 1
-!ENDIF
+#!IFNDEF JPEG12_SUPPORTED
+#JPEG12_SUPPORTED = 1
+#!ENDIF
 
 #if using an external jpeg library uncomment the following lines
 #JPEG_EXTERNAL_LIB = 1
